packages/Chips/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+import withUtils from "@blaze-react/utils";
+import React, { Fragment, ReactElement, useState } from "react";
+import ChipAvatar from "./ChipAvatar";
+import ChipIcon from "./ChipIcon";
+import ChipLabel from "./ChipLabel";
+
+interface IChipsProps {
+  children: JSX.Element | JSX.Element[];
+  modifiers: string[];
+  action?: () => void;
+  utils: {
+    classNames: (className: string | object, classNames?: object) => string;
+  };
+}
+const Chips = withUtils(
+  ({
+    children,
+    modifiers,
+    action,
+    utils: { classNames }
+  }: IChipsProps): JSX.Element => {
+    const [showChip, setChip] = useState<boolean>(true);
+
+    const handleRemoveChip = (): void => setChip(false);
+
+    const chipClassName: string = classNames("chip", {
+      [modifiers
```

Comment:
This code```modifiers.map(modifier => `chip--${modifier}`).join(" ") ``` should by in a function and you can call this fn inside square brackets. ```[getModifiers()]: condition``` try to do the code the most readable possible.